src/Provider/Provider.Web/Controllers/ExternalLinksController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Esfa.Recruit.Provider.Web.Configuration;
+using Esfa.Recruit.Provider.Web.Configuration.Routing;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Esfa.Recruit.Provider.Web.Controllers
+{
+	public class ExternalLinksController : Controller
+    {
+        private readonly ProviderApprenticeshipsLinkHelper _linkHelper;
+
+        public ExternalLinksController(ProviderApprenticeshipsLinkHelper linkHelper)
+        {
+            _linkHelper = linkHelper;
+        }
+
+        [HttpGet(RoutePaths.AccountRoutePath + "/account-home", Name = RouteNames.Dashboard_Account_Home)]
```

Comment:
Can we put `RoutePaths.AccountRoutePath` as a class level attribute?

Reply:
👍